### DISCUSSION TITLE
Update and improve the  Getting Started guides

### DISCUSSION THREAD

**Discussion by Zerotask** [photonstorm/phaser#5906] [Feature Requests]
The current website doesn't provide an easy and up-to-date guide on how to create a basic game with Phaser.
I have to click on a button which says I have to download it from GitHub. For a JS project I'm used to see a simple npm install command (which shows the page then).
Then the JavaScript guide leads me to a small page where I have to decide for version 3 or 2. As a beginner I don't know the difference and want to use the newest. After that I see a lot of text about web server and stuff. I don't understand why it's not simple done using NodeJS. Shouldn't be a big deal.
Then there is a 3 years old page about editors with a too big Sublime icon and Visual Studio Code (which most people use nowadays) is just in the middle. After that page it shows me - again -something about downloading phaser. But I already downloaded it, because that was step 1. Then finally, after clicking through 10 pages I see actual code of Phaser. This should be on the first getting started page.

The TypeScript guide is from 2013 and completely outdated. Additionally it uses Visual Studio which you normally don't use for a JS project nowadays. You'd use Visual Studio Code or Webstorm. So you can't really use the TypeScript guide.

The complete getting started section is not really good in my opinion and should be updated. This would make it much easier for new people to start with Phaser.

**Comment by HarmHoeksema** [comment]
I think there is already an ongoing plan to revamp the website.
